.ci/.e2e-tests.yaml

```diff
@@ -27,11 +27,6 @@ SUITES:
       - name: "APM Integration"
         pullRequestFilter: " && ~debian"
         tags: "apm_server"
-      - name: "Endpoint Integration"
```

Comment:
We could have skipped only the affected scenarios, but Eric mentioned endpoint does not support docker at this moment, so it seems safer to skip the entire test suite.

Reply:
acknowledged.  @ferullo fyi - we're pulling these out.  but hopefully it is only temporary.  We should reenable against Linux + Windows (and macOS) after features from here are implemented wherein we'll use real VMs for the Agent install:
https://github.com/elastic/e2e-testing/issues/1340

Reply:
Yes, the tests are still there, so it's totally fine to execute them locally against containers. On CI, after #1340 lands, we will be able to install the agent the real VM represented by the CI worker, and then we can reenable these endpoint tests.

OTOH, the big majority of the tests usually pass, even not supported in Docker containers 🤔